sunpy/time/tests/test_time.py

```diff
@@ -150,14 +150,6 @@ def test_parse_time_individual_numpy_datetime():
     assert dt == Time('2005-02-01', format='isot')
 
 
-def test_parse_time_numpy_datetime_timezone():
```

Comment:
I removed this test as numpy plan to remove parsing timezone aware datetimes, so.